qemu/tests/x2avic_test.py

```diff
@@ -0,0 +1,45 @@
+from avocado.utils import process
+from virttest import env_process
+from provider.cpu_utils import check_cpu_flags
+
+
+def run(test, params, env):
+    """
+    avic test:
+    1) Turn on avic on Genoa host
+    2) Verify kernel x2AVIC is enabled on host
+    3) Boot a guest
+    4) Check 'x2apic' enabled inside guest
+    5) Then run the follow linux perf utility to verify VMEXIT counter on host
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    flags = params["flags"]
+    check_cpu_flags(params, flags, test)
+
+    params["start_vm"] = "yes"
+    vm = env.get_vm(params["main_vm"])
+    env_process.preprocess_vm(test, params, env, vm.name)
```

Comment:
What resources need to be prepared here?

Reply:
> What resources need to be prepared here?

updated to `vm.create()`, Thanks.